Pages/registerPage.js

```diff
@@ -0,0 +1,227 @@
+    }
+
+    validateInvalidRegistrationMessage(invalidRegistrationMessage){
+        console.log("This method validates invalid Registration message");
+        return expect(invalidRegistrationMessage).toBe(data.usedEmailMessage);
+    }
+
+    validateRequiredFirstName(invalidFirstName){
+        console.log("This method validates invalid (empty) First Name");
+        return expect(invalidFirstName).toBe(data.firstNameRequiredMessage);
+    }
+
+    validateInvalidFirstName(invalidFirstName){
+        console.log("This method validates invalid (special characters) First Name");
+        return expect(invalidFirstName).toBe(data.firstNameWithSpecialCharacters);
+    }
+
+    validateInvalidConfirmPassword(invalidConfirmPassword){
+        console.log("This method validates invalid (empty) Confirm Password");
+        return expect(invalidConfirmPassword).toBe(data.invalidConfirmPasswordValidationMessage);
+    }
+    validateInvalidEmail(invalidEmail){
+        console.log("This method validates invalid (empty) Email");
+        return expect(invalidEmail).toBe(data.emailRequiredMessage);
+    }
+
+    validateRequiredLastName(invalidLastName){
+        console.log("This method validates invalid (empty) Last Name");
+        return expect(invalidLastName).toBe(data.lastNameRequiredMessage);
+    }
+
+    validateInvalidLastName(invalidLastName){
+        console.log("This method validates invalid (special characters) Last Name");
+        return expect(invalidLastName).toBe(data.lastNameWithSpecialCharacters);
+    }
+
+    validateInvalidPassword(invalidPassword){
+        console.log("This method validates invalid (empty) Password");
+        return expect(invalidPassword).toBe(data.passwordValidationMessage);
+    }
+
+    validateRegisterPageURL(URL){
+        console.log("This method validated entered URL");
+        return expect(URL).toBe(data.registerpageLink);
+    }
+
+    clickOnRegisterButton(){
+        console.log("This method clicks on Register button");
+        return this.registerButton.click();
+    }
+
+    enterConfirmPassword(data){
+        console.log("This method enters data to Confirm Password field");
+        return this.confirmPasswordField.sendKeys(data);
+    }
+
+    enterPassword(data){
+        console.log("This method enters data to Password field");
+        return this.passwordField.sendKeys(data);
+    }
+
+    enterEmail(){
+        console.log("This method sends data to email field");
+        return this.emailField.sendKeys(this.makeRandomEmail());
+    }
+    enterEmailWithoutDomain(emailData){
+        console.log("This method sends email without domain to email field");
+        return this.emailField.sendKeys(this.makeRandomEmailWithoutdomain(emailData));
+    }
+
+    enterDataEmail(email2){
+        // This method had to be developed this way, because data sent through registration is used for the login, and data is static
+        console.log("This method sends us@mail.com to email field");
```

Comment:
Should it be "console.log(`This method sends ${email2} to email field`);" here ?
What is "us@mail.com" ?

Reply:
I added template literals in this console.log(), so that this part of code is dynamic, in case that I change var email2 in data.js for tests to pass.